test/integ_tests/test_all_notebooks.py

```diff
@@ -178,6 +178,54 @@ def execute_with_mocks(tb, mock_level, path_to_utils, path_to_mocks):
         run=False,
         before=0,
     )
+
+    # Uncomment all test sections in the notebook
+    for i, cell in enumerate(tb.cells):
+        if cell.get("cell_type") == "code" and "source" in cell:
+            source = cell["source"]
+            if "# UNCOMMENT_TO_RUN" in source:
+                # Uncomment the test section
+                modified_source = uncomment_test_section(source)
+                tb.cells[i]["source"] = modified_source
+
+    # Execute the notebook with the uncommented test sections
     tb.execute()
     test_mocks = SourceFileLoader("notebook_mocks", path_to_mocks).load_module()
     test_mocks.post_run(tb)
+
+
+def uncomment_test_section(source):
+    """Uncomment sections marked with # UNCOMMENT_TO_RUN."""
+    descriptive_comment = re.compile(r"^\s*##\s")
+    regular_comment = re.compile(r"^\s*#\s?")
+
+    lines = source.splitlines()
+    result = []
+    uncomment_mode = False
+
+    for line in lines:
+        stripped_line = line.strip()
+
+        if stripped_line == "# UNCOMMENT_FOR_TEST":
```

Comment:
I don't see where you have this string in the notebook. Should it be the same as the string above, `"# UNCOMMENT_TO_RUN"`? (If so, maybe define that string at the file scope, to use in both places?)

Reply:
Yes, I am good to change that to descriptive comment.

Reply:
@rmshaffer Oh wait, I missed out changing the tag here. Let me fix that.